server/pulp/server/async/tasks.py

```diff
@@ -764,20 +764,19 @@ def on_failure(self, exc, task_id, args, kwargs, einfo):
         :param kwargs:  Original keyword arguments for the executed task.
         :param einfo:   celery's ExceptionInfo instance, containing serialized traceback.
         """
+        original_formatted_traceback = traceback.format_exc()
         if isinstance(exc, PulpCodedException):
             _logger.info(_('Task failed : [%(task_id)s] : %(msg)s') %
                          {'task_id': task_id, 'msg': str(exc)})
             _logger.debug(traceback.format_exc())
```

Comment:
@bmbouter since we have introduced this variable, can we make sure we also use it here? :)